### PR TITLE
More Racha Methods

### PR DESCRIPTION
**Pull request by edukatiau**
Created Join and Leave method of Racha

### CODE REVIEW

**Review by edukatiau — COMMENTED** [review]
test commit

File: src/main/java/ifsul/agileproject/rachadinha/mapper/RandomCodeGenerator.java
Comment (DiegoDexheimer) [inline src/main/java/ifsul/agileproject/rachadinha/mapper/RandomCodeGenerator.java:1]: Isn't it better to move the file to the utilities package?

Comment (edukatiau) [reply src/main/java/ifsul/agileproject/rachadinha/mapper/RandomCodeGenerator.java:1]: For me, how this util class is used just in the mapper, doesnt make necessary create a utils package. But if it's better, we can do it.

Comment (edukatiau) [reply src/main/java/ifsul/agileproject/rachadinha/mapper/RandomCodeGenerator.java:1]: (At least i don't see it being used again in the future)

Comment (DiegoDexheimer) [reply src/main/java/ifsul/agileproject/rachadinha/mapper/RandomCodeGenerator.java:1]: Hmm Ok 👍
7vhM7Up

**Review by DiegoDexheimer — APPROVED** [review]

**Pull request merged**